assets/keymaps/vim.json

```diff
@@ -400,7 +400,8 @@
   {
     "context": "Editor && vim_mode == visual && !VimWaiting && !VimObject",
     "bindings": {
-      "u": "editor::Undo",
```

Comment:
It looks like removing this binding is causing problems. When testing out this change, hitting `u` in normal mode correctly undos back until it reaches the the `ConvertToLowerCase` operation, which places it in visual mode, which causes `u` to no longer undo.

I _think_ the fix here is to not enter visual mode when cycling through undos.

Reply:
I think you're right. There's a fairly big mismatch right now between how vim does undo and how zed does undo.

That said, I think we should probably move in the direction of "more compatible", and so I'm happy to merge this without figuring that out *yet* (though it definitely bumps it up the priority list!).

If you'd like to pair on that (or another vim issue), feel free to book time with me:
https://calendly.com/conradirwin/pairing